src/coreComponents/constitutive/capillaryPressure/TableCapillaryPressure.cpp

```diff
@@ -84,11 +85,12 @@ void TableCapillaryPressure::postProcessInput()
   else if( numPhases == 3 )
   {
     GEOSX_THROW_IF( m_wettingIntermediateCapPresTableName.empty() || m_nonWettingIntermediateCapPresTableName.empty(),
-                    GEOSX_FMT(
-                      "{}: for a three-phase flow simulation, we must use {} to specify the capillary pressure table for the pair (wetting phase, intermediate phase), and {} to specify the capillary pressure table for the pair (non-wetting phase, intermediate phase)",
-                      getFullName(),
-                      viewKeyStruct::wettingIntermediateCapPresTableNameString(),
-                      viewKeyStruct::nonWettingIntermediateCapPresTableNameString()  ),
+                    GEOSX_FMT( string( "{}: for a three-phase flow simulation, we must use {} to specify the capillary pressure table " ) +
+                               string( "for the pair (wetting phase, intermediate phase), and {} to specify the capillary pressure table " ) +
+                               string( "for the pair (non-wetting phase, intermediate phase)" ),
```

Comment:
No need to break format string literal apart, you should be able to format it like this (I hope uncrustify doesn't mess it up): 
```suggestion
                    GEOSX_FMT( "{}: for a three-phase flow simulation, we must use {} to specify the capillary pressure table "
                               "for the pair (wetting phase, intermediate phase), and {} to specify the capillary pressure table "
                               "for the pair (non-wetting phase, intermediate phase)" ,
```

Reply:
I modified these error messages as you suggested.